notebooks/ullyses-timeseries.ipynb

```diff
@@ -0,0 +1,639 @@
+    "        plt.xlabel(\"Wavelength [$\\mathrm{\\AA}$]\")\n",
+    "        plt.ylabel(\"Flux\\n[ergs/s/cm^2/$\\mathrm{\\AA}$]\")\n",
+    "    plt.legend(handles=plot_legends)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot_x1ds(x1ds)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It's a little hard to see any variability here, so let's zoom in on the C IV line."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot_x1ds(x1ds, xlim=(1545, 1555))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From this plot, we can see there seems to be two different flux states for this star, corresponding to the different observational epochs in 2014 and 2020. Because of this, the ULLYSES team cannot coadd all of the available G160M data. Instead, we create a time-series spectrum for this target."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"sec2\"></a>\n",
+    "## Creating a time-series spectrum"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"sec21\"></a>\n",
+    "### Time-series YAML files"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To create a TSS, we need to specify which grating and datasets should be used. This is done by creating a YAML file with the relevant info.\n",
+    "YAML files are convenient, human-readable files that act as configuration setups. The ULLYSES team creates timeseries YAML files for all targets that have been noted to be variable. \n",
+    "\n",
+    "Let's take a look at the CVSO-109 timeseries YAML file now. All timeseries YAML files can be found in the [`ullyses_utils` package](https://github.com/spacetelescope/ullyses-utils/tree/main/src/ullyses_utils/data/timeseries)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Find your local installation of the ullyses_utils package\n",
+    "utils_dir = ullyses_utils.__path__[0]\n",
+    "# Get the path to the timeseries YAML files\n",
+    "yamls = glob.glob(os.path.join(utils_dir, f\"data/timeseries/*cvso-109*.yaml\"))\n",
+    "yamls.sort()\n",
+    "pprint.pprint(yamls)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There is just one YAML for CVSO-109 COS data, so let's open it and take a look."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Use the ULLYSES convenience function to read the YAML file\n",
+    "yaml = yamls[0]\n",
+    "config = read_config(yaml)\n",
+    "print(f\"{os.path.basename(yaml)}:\")\n",
+    "print(config)"
```

Comment:
`pprint.pprint(config)` to make a little easier to read?

Reply:
I did that but it alphabetizes it and I want it in the order it is in the yaml... I made the print prettier manually :D